src/System.Text.Json/src/System/Text/Json/Linq/JArray.cs

```diff
@@ -200,6 +200,25 @@ public JArray(IEnumerable<decimal> values) : this()
             }
         }
 
+        /// <summary>
+        ///   Parses a string representing JSON document into a <see cref="JArray"/>.
```

Comment:
Maybe we should change the word "document" in `JNode.Parse` (and `JObject.Parse` and `JArray.Parse`) documentation, so that users won't associate it with `JsonDocument`?